### PR TITLE
Mk add new data sources

### PR DESCRIPTION
**Pull request by mkangia**
1. Update test data sources to include notification related fields for new report to be added
2. Update/Add person data source to check for hiv status updates

best reviewed 🐟 / 🐟

### CODE REVIEW

File: custom/enikshay/ucr/data_sources/test_2b_v2.json [hunk 1]
```diff
@@ -0,0 +1,3377 @@
+{
+    "domains": [
+        "enikshay",
```
Comment (emord) [inline custom/enikshay/ucr/data_sources/test_2b_v2.json]: this is called "test_2b_v2" If it's in testing does it need to be in the enikshay domain, or can it be kept in the QA domain for now and added to enikshay later?

File: custom/enikshay/ucr/data_sources/test_2b_v2.json [hunk 2]
```diff
@@ -0,0 +1,3377 @@
...
+        "sheel-enikshay",
+        "enikshay-reports-qa",
+        "enikshay-performance-test"
+    ],
+    "server_environment": [
+        "enikshay",
+        "softlayer"
```
Comment (emord) [inline custom/enikshay/ucr/data_sources/test_2b_v2.json]: I don't think these need to be added to softlayer anymore. Is there any testing going on in that cluster

**Review by emord — COMMENTED** [review]
I don't have any context for the calculations. You'll also need to add this file path to settings.py

File: custom/enikshay/ucr/data_sources/test_2b_v2.json (hunk 2 above)
Comment (mkangia) [reply custom/enikshay/ucr/data_sources/test_2b_v2.json]: sorry. removing this.

File: custom/enikshay/ucr/data_sources/test_2b_v2.json (hunk 1 above)
Comment (mkangia) [reply custom/enikshay/ucr/data_sources/test_2b_v2.json]: its needs to run on the real data. so should be under enikshay?

Comment (proteusvacuum) [reply custom/enikshay/ucr/data_sources/test_2b_v2.json]: it's the `test` case type :smile:

**Review by proteusvacuum — CHANGES_REQUESTED** [review]
I think you need to add this to `settings.py`?

File: custom/enikshay/ucr/data_sources/test_2b_v2.json (hunk 1 above)
Comment (mkangia) [reply custom/enikshay/ucr/data_sources/test_2b_v2.json]: oh ya.. test is the case type :)

**Comment by mkangia** [comment]
@proteusvacuum thank you for the settings miss. Added.

**Review by nickpell — APPROVED** [review]

**Review by proteusvacuum — APPROVED** [review]

**Pull request merged**